src/lazy/text/value.rs

```diff
@@ -40,41 +41,47 @@ impl<'a> Debug for MatchedRawTextValue<'a> {
 // ===== Version-specific wrappers =====
 //
 // These types provide Ion-version-specific impls of the LazyRawValue trait
-#[derive(Copy, Clone, Debug)]
-pub struct LazyRawTextValue_1_0<'data> {
+#[derive(Copy, Clone)]
+pub struct LazyRawTextValue<'data, E: TextEncoding<'data> + Copy> {
     pub(crate) matched: MatchedRawTextValue<'data>,
+    spooky: PhantomData<E>,
 }
```

Comment:
🗺️ Changes in this file stem from this change, which makes `LazyRawTextValue` generic over the text encoding version. Below, you can see that `LazyRawTextValue_1_0` and `LazyRawTextValue_1_1` are now just type aliases.